src/View/Helper/TimeHelper.php

```diff
@@ -373,7 +370,7 @@ public function format($date, $format = null, $invalid = false, $timezone = null
      * Returns a formatted date string, given either a Datetime instance,
      * UNIX timestamp or a valid strtotime() date string.
      *
-     * @param int|string|\DateTime $date UNIX timestamp, strtotime() valid string or DateTime object
+     * @param int|string|\DateTimeInterface|null $date UNIX timestamp, strtotime() valid string or DateTime object
```

Comment:
If this officially supports null, shouldn't that the be the default for $date?  Feels weird as it returns $invalid.

Reply:
The null check is only a safeguard. I had to add it to the docblock too to prevent static analyzers from complaining.